### PR TITLE
Auth: do not escape parameters in rendered IRMA contracts

### PR DESCRIPTION
**Pull request by reinkrul**
Fixes https://github.com/nuts-foundation/nuts-node/issues/1931

Also apply on master.

### CODE REVIEW

File: auth/contract/contract_test.go [hunk 1]
```diff
@@ -222,4 +223,12 @@ func TestParseContractString(t *testing.T) {
 		assert.NotNil(t, err)
 		assert.True(t, errors.Is(err, ErrInvalidContractText))
 	})
+	t.Run("ok", func(t *testing.T) {
```
Comment (stevenvegt) [inline auth/contract/contract_test.go:226]: I would make this explicitly an ampersand test and rename it to something like: "ok - it parses a contract text containing ampersand characters"

File: auth/contract/contract_test.go [hunk 2]
```diff
@@ -222,4 +223,12 @@ func TestParseContractString(t *testing.T) {
...
+		rawText := "NL:BehandelaarLogin:v1 Ondergetekende geeft toestemming aan Demo EHR om namens Toon & Zoonen en ondergetekende het Nuts netwerk te bevragen. Deze toestemming is geldig van dinsdag, 1 oktober 2019 13:30:42 tot dinsdag, 1 oktober 2019 14:30:42."
+		signedContract, err := ParseContractString(rawText, StandardContractTemplates)
+
+		require.NoError(t, err)
+		require.NotNil(t, signedContract)
+		assert.Equal(t, "", signedContract.Params[""])
```
Comment (stevenvegt) [inline auth/contract/contract_test.go:232]: why checking the empty string? Would make sense to check if the params contain the actual "Toon & Zoon" information in the `legal_entity`.

**Review by stevenvegt — CHANGES_REQUESTED** [review]

**Comment by stevenvegt** [comment]
Tests are failing because the e2e tests now include external crypto storage, but the V5 branch doesn't.

**Review by stevenvegt — APPROVED** [review]

**Comment by reinkrul** [comment]
I tested it with the IRMA app, using the current v5.0.9 version the IRMA app shows the contract text with `&amp;`, with this patch it shows it as `&`. Looks OK.

**Comment by gerardsn** [comment]
manual run of e2e tests is successful

**Pull request merged**